### ISSUE TITLE
[FEATURE] Small formatting/linting Jinja issues

### ISSUE THREAD

**Issue by rickdonato** [djlint/djLint#262]
## Feature Request
On testing djlint some more, I found a few small areas that weren't covered when linting/formatting Jinja templates.

Additional spacing between condition and variable not detected.
```
{% if   abc == 101 %}
       ^
```

Enclosing quotes are not standardized. I would think they would need to be changed to `"` or `'`. Not sure if there is a Jinja standard for which. Looking at https://sites.google.com/a/chromium.org/dev/developers/jinja they seem it use `'`
```
{% if   abc == "101" %}
{% endif %}
{% if   abc == '101' %}
{% endif %}
```

Thanks again for the great tool. 

Rick

**Comment by monosans** [comment]
Both halves are handled now, Rick.

The spacing was already covered in released versions: `T032` reports it and `--reformat` condenses it, so `{% if   abc == 101 %}` comes back as `{% if abc == 101 %}`.

The quotes were the gap. In v1.45.0, `--quote-style` reaches conditions too, so a file no longer spells the same string both ways:

```console
$ djlint page.html --reformat --profile jinja
```

```jinja
{% if abc == "101" %}{% endif %}
{% if abc == "101" %}{% endif %}
```

Both of your snippets converge. `--quote-style single` gives `'101'` instead; `double` is the default. `{% elif %}` and `{% elsif %}` are covered as well.

A string is left as written when rewriting it would need escaping, so `{% if a == 'say "hi"' %}` keeps its single quotes, and a tag inside an html attribute keeps whatever the attribute's own quotes allow. A tag shown as text inside `{% raw %}` or `{% verbatim %}` is left alone too.

Released in v1.45.0. Thanks for the clear report.
